api/doc/openapi.yaml

```diff
@@ -504,6 +504,147 @@ paths:
           $ref: '#/components/responses/415'
         "500":
           $ref: '#/components/responses/500'
+  /tokens/{TokenId}:
+    get:
+      description: "Returns Token by Id"
+      operationId: getTokenById
+      parameters:
+        - name: "TokenId"
+          in: path
+          required: true
+          description: "Token Id, has format `${address}:${collection_name}:${token_name}`"
+          schema:
+            $ref: "#/components/schemas/TokenId"
+      responses:
+        "200":
+          description: "OK"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Token"
+        '404':
+          $ref: '#/components/responses/404'
+        '400':
+          $ref: '#/components/responses/400'
+        '500':
+          $ref: '#/components/responses/500'
+  /token_metadata/{tokenId}:
+    get:
+      description: "Returns TokenMetaData by Id"
+      operationId: getTokenMetaDataById
```

Comment:
Metadata and only MetaData (in addition) if there's an issue with another library